src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/MakeAmendmentOfSpecificMethodologyAuthorizationHandlers.cs

```diff
@@ -0,0 +1,67 @@
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Security;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using IdentityServer4.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers.AuthorizationHandlerUtil;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Security.SecurityClaimTypes;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers
+{
+    public class MakeAmendmentOfSpecificMethodologyRequirement : IAuthorizationRequirement
+    {
+    }
+
+    public class MakeAmendmentOfSpecificMethodologyAuthorizationHandler
+        : AuthorizationHandler<MakeAmendmentOfSpecificMethodologyRequirement, Methodology>
+    {
+        private readonly IUserPublicationRoleRepository _userPublicationRoleRepository;
+
+        public MakeAmendmentOfSpecificMethodologyAuthorizationHandler(
+            IUserPublicationRoleRepository userPublicationRoleRepository)
+        {
+            _userPublicationRoleRepository = userPublicationRoleRepository;
+        }
+
+        protected override async Task HandleRequirementAsync(
+            AuthorizationHandlerContext context,
+            MakeAmendmentOfSpecificMethodologyRequirement requirement,
+            Methodology methodology)
+        {
+            // Amendments can only be created from Methodologies that are already publicly-accessible.
+            if (!methodology.PubliclyAccessible)
+            {
+                return;
+            }
+
+            // Any user with the "MakeAmendmentsOfAllMethodologies" Claim can create an amendment of a
+            // publicly-accessible Methodology.
+            if (SecurityUtils.HasClaim(context.User, MakeAmendmentsOfAllMethodologies))
+            {
+                context.Succeed(requirement);
+                return;
+            }
+
+            // If the user has a Publication Owner role on a Publication that uses this Methodology, they can create 
+            // an Amendment of this Methodology.
+            if (methodology.Publications.IsNullOrEmpty())
```

Comment:
I think it's worth bring this back up in discussion because so far we've said the amending ability will be limited to a user that has the Owner role on the Publication which the methodology belongs to, rather than Owner role on any Publication using the Methodology.

If we based it on any publication using the methodology then there will be a loophole where any publication owner can in theory get access to amend any methodology just by adopting it.

The same is true for the updating permission which is limited to users who can update the latest release of the publication which the methodology belongs to. The publication owner and their release team would be able to update any methodology that the publication owner decides to adopt.

Instead, the publication owner and release team will either ask the relevant team to make the change or request to become publication owner on the publication / contributor on that publications latest release.

Reply:
Yup that sounds good.  I guess we don't have the model changes in yet to support this, so I could just put a TODO against this with the ticket ref of the model change ticket?

Reply:
I think from the SOW4 branch we do have that change so you could either target SOW4 or wait until this is merged via dev back into it and keep this ticket open to make further changes at that point in time?

Reply:
Yup, will do the latter then, thanks!

Reply:
Added a TODO to tackle after merge with SOW4 branch